### DISCUSSION TITLE
Recovering my NextCloud Box - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#42] [General]
# Recovering my NextCloud Box - Alan Pope's blog

NextCloud Box I just stumbled on an old NextCloud Box in my loft. It’s a quiet Sunday in the house, so I thought I’d see if it still works, and if there’s any data on it. I’m pretty sure I did use it for a while, so there must be something on it.
Here’s my NextCloud Box in a cardboard box labelled “NextCloud Box”.
Here’s what the NextCloud box looks like once installed.

https://popey.com/blog/2023/10/recovering-my-nextcloud-box/

**Comment by Daviey** [comment]
I'm *really* shocked with the routing of the USB power.  For a device like this, I can't understand why they didn't power it from a regulated 5v supply directly to the power GPIO pins!  This is super fugly.

**Reply by popey** [reply]
It's pretty janky, to be sure! :D
